openc3-cosmos-cmd-tlm-api/app/controllers/model_controller.rb

```diff
@@ -48,6 +54,9 @@ def show
     else
       render json: @model_class.get(name: params[:id], scope: params[:scope])
     end
+  rescue StandardError => error
+    render json: { status: 'error', message: error.message }, status: 500
+    OpenC3::Logger.error(error.formatted)
```

Comment:
Logger errors cause messages in CmdTlmServer - Probably don't want that here.   Change to using rails logger.